docs/migration-guides/1.1-1.2.md

```diff
@@ -0,0 +1,29 @@
+# Migrate from v1.1 to v1.2
+
+## Event Bus
+
+The 1.2 release updates the Knative eventing from version 0.4 to 0.5. This brings in considerable changes to the Knative eventing CRDs which cannot be automatically done in Kyma update. To fix this, perform the following post-update steps.
```

Comment:
```suggestion
The 1.2 release updates the Knative eventing from version 0.4 to 0.5. This brings in considerable changes to the Knative eventing CRDs which cannot be created automatically during Kyma update. To fix this, perform the following post-update steps.
```

Can we have `created` instead of `done`? I don't really know if we can "do" a CRD :)